{
	"documenttitle": {
		"text": "Red Hat Security Advisory: Red Hat AMQ Streams 2.2.0 release and security update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2022:6819"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2022-10-05T14:30:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2022-10-05T14:30:00Z",
		"CurrentReleaseDate": "2022-10-05T14:30:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.9.3",
			"Date": "2022-10-05T21:12:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nRed Hat AMQ Streams 2.2.0 is now available from the Red Hat Customer Portal.\n\nRed Hat Product Security has rated this update as having a security impact of Important. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nRed Hat AMQ Streams, based on the Apache Kafka project, offers a distributed backbone that allows microservices and other applications to share data with extremely high throughput and extremely low latency. \n\nThis release of Red Hat AMQ Streams 2.2.0 serves as a replacement for Red Hat AMQ Streams 2.1.0, and includes security and bug fixes, and enhancements.\n\nSecurity Fix(es):\n\n* kafka: Unauthenticated clients may cause OutOfMemoryError on brokers (CVE-2022-34917)\n\n* jackson-databind: denial of service via a large depth of nested objects (CVE-2020-36518)\n\n* netty: world readable temporary file containing sensitive data (CVE-2022-24823)\n\n* com.google.code.gson-gson: Deserialization of Untrusted Data in com.google.code.gson-gson (CVE-2022-25647)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2022 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Important",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2022:6819",
				"Description": "https://access.redhat.com/errata/RHSA-2022:6819"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#important",
				"Description": "https://access.redhat.com/security/updates/classification/#important"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=jboss.amq.streams&version=2.2.0",
				"Description": "https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=jboss.amq.streams&version=2.2.0"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2",
				"Description": "https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "jackson-databind: denial of service via a large depth of nested objects",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-08-13T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-36518",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:6819"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://github.com/advisories/GHSA-57j2-w4cx-62h2",
						"Description": "https://github.com/advisories/GHSA-57j2-w4cx-62h2"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-36518",
						"Description": "CVE-2020-36518"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2064698",
						"Description": "bz#2064698: CVE-2020-36518 jackson-databind: denial of service via a large depth of nested objects"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-03-16T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "netty: world readable temporary file containing sensitive data",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-05-06T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-24823",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:6819"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-24823",
						"Description": "CVE-2022-24823"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2087186",
						"Description": "bz#2087186: CVE-2022-24823 netty: world readable temporary file containing sensitive data"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-05-17T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "com.google.code.gson-gson: Deserialization of Untrusted Data in com.google.code.gson-gson",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-05-01T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-25647",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:6819"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-25647",
						"Description": "CVE-2022-25647"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2080850",
						"Description": "bz#2080850: CVE-2022-25647 com.google.code.gson-gson: Deserialization of Untrusted Data in com.google.code.gson-gson"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-05-02T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "Kafka: Unauthenticated clients may cause OutOfMemoryError on brokers",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-09-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-34917",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nBefore applying the update, back up your existing installation, including all applications, configuration files, databases and database settings, and so on.\n\nThe References section of this erratum contains a download link (you must log in to download the update).\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:6819"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-34917",
						"Description": "CVE-2022-34917"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2130018",
						"Description": "bz#2130018: CVE-2022-34917 Kafka: Unauthenticated clients may cause OutOfMemoryError on brokers"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Important"
				}
			},
			"DiscoveryDate": "2022-09-21T00:00:00Z",
			"productstatuses": {
				"status": {
					"ProductID": null
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {}
}
